test/cypress/integration/39-recurring-contributions.test.js

```diff
@@ -1,31 +1,21 @@
-import mockRecaptcha from '../mocks/recaptcha';
-
 describe('Recurring contributions', () => {
+  let user;
+
   before(() => {
-    const visitParams = { onBeforeLoad: mockRecaptcha };
-    cy.login().then(() => {
-      cy.visit('/apex/contribute/sponsors-470/checkout', visitParams);
+    cy.signup({ redirect: '/apex/contribute/sponsors-470/checkout' }).then(u => {
```

Comment:
When editing the data (and possible), it's better to use `singup` to create a new user rather than relying on the default `testusercollective`. More info: https://docs.opencollective.com/help/contributing/development/testing-with-cypress#tests-independence